src/message_buffer.cpp

```diff
@@ -3,24 +3,90 @@
 #include <iomanip>
 #include <sstream>
 
-namespace quicr::messages {
+namespace quicr {
+uintVar_t
+to_varint(uint64_t v)
```

Comment:
This is just an assignment to int.  Do we want a "real" variable length integer?  QUIC defines one for unsigned int.  Cullen defined a different one for GameState, and GSE has both signed and unsigned.